app/assets/stylesheets/shared/bootstrap-extensions.css.scss

```diff
@@ -10,3 +10,35 @@
     }
   }
 }
+
+#chat-with-us {
```

Comment:
Do you think these styles belongs to `bootstrap-extensions.css.scss`? Are we overriding bootstrap's default?